Source/WebCore/style/FilterOperationsBuilder.cpp

```diff
@@ -192,7 +192,9 @@ std::optional<FilterOperations> createFilterOperations(const Document& document,
         }
     }
 
-    return operations;
+    operations.shrinkToFit();
```

Comment:
Somehow I thought that move already did this?

Reply:
Vector move is just really a pointer swap, no allocation changes.